### PR TITLE
Added test cases for short term pipeline scripts

### PR DESCRIPTION
**Pull request by YakMan101**
Added a range of test cases for short term pipeline scripts. Changed some docstrings too. 

I wasn't able to test any functions that use calls to the RDS database as I couldn't find a way to properly patch pymssql.connect. It would always try to connect to RDS with credentials i can't give it without exposing sensitive info

### CODE REVIEW

File: short_term/database_handler.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 from os import environ as ENV
 
-from pymssql import connect
+from pymssql import connect  # pylint: disable=E0611
```
Comment (Leon-JS02) [inline short_term/database_handler.py:6]: What's `E0611`?

**Review by Leon-JS02 — APPROVED** [review]
Nice optimisations with async

File: terraform/sample_image_short/main.py
Comment (olliejth) [inline terraform/sample_image_short/main.py:1]: What does this pylint command disable?

File: short_term/extract_short.py [hunk 2]
```diff
@@ -53,7 +63,15 @@ def extract_recordings() -> str:
     """Gets the recordings and turns them into json data.
     The main function to be called from the pipeline."""
 
-    api_data = get_plant_data()
+    plant_ids = list(range(1, 50 + 1))
```
Comment (mehrdadhalali) [inline short_term/extract_short.py:59]: Why not 51

**Review by mehrdadhalali — APPROVED** [review]

**Review by olliejth — APPROVED** [review]

File: short_term/database_handler.py (hunk 1 above)
Comment (YakMan101) [reply short_term/database_handler.py:6]: Disable 'connect doesn't exist' for pymssql library errorwhen it clearly does exist

**Review by olliejth — APPROVED** [review]
Good comprehensive testing

**Pull request merged**